### PR TITLE
Additional column detection, pdf download for papers and fact sheets

### PR DESCRIPTION
**Pull request by spodgorny9**

### CODE REVIEW

File: elm/utilities/parse.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-def is_multi_col(text, separator="    "):
+def is_multi_col(text, separator="    ", threshold_ratio=0.35):
```
Comment (grantbuster) [inline elm/utilities/parse.py:16]: Why implement this in addition to `PDFtoTXT.is_double_col()`? Why not remove `is_double_col` from PDFtoTXT or have is_multi_col be a static method in PDFtoTXT? (latter is probably prefered).

**Review by grantbuster — CHANGES_REQUESTED** [review]

File: elm/utilities/parse.py (hunk 1 above)
Comment (spodgorny9) [reply elm/utilities/parse.py:16]: This was mirroring the previous set up where `PDFtoTXT.is_double_col()` uses `is_multi_col()` but if there is no reason to preserve that structure I can set up the static method.

Comment (grantbuster) [reply elm/utilities/parse.py:16]: oh yeah and `is_multi_col` is in a utilities module.... ehhhh i guess it doesnt really matter haha

Comment (grantbuster) [reply elm/utilities/parse.py:16]: I think you should remove is_double_col from PDFtoTXT and just call the utility function where required without it being attached to PDFtoTXT

Comment (spodgorny9) [reply elm/utilities/parse.py:16]: Sounds good, will do!

**Review by grantbuster — APPROVED** [review]

**Pull request merged**